corehq/apps/analytics/tasks.py

```diff
@@ -393,3 +396,9 @@ def _log_response(data, response):
         logger.error(message)
     else:
         logger.debug(message)
+
+
+def _get_ab_test_properties(user):
```

Comment:
I think we should add a property slug to this function so that not all users are permanently in an A or B group when we test other stuff (unless you wanted it that way)